keiko-core/src/main/kotlin/com/netflix/spinnaker/q/Queue.kt

```diff
@@ -93,12 +98,17 @@ interface Queue {
    */
   val deadMessageHandlers: List<DeadMessageCallback>
 
+  /**
+   * Denotes a queue implementation capable of processing multiple messages per poll.
+   */
+  val canPollMany: Boolean
+
   companion object {
     /**
      * The maximum number of times an un-acknowledged message will be retried
      * before failing permanently.
      */
-    val maxRetries: Int = 5
+    val maxRetries: Int = 10
```

Comment:
Bumped this for why? Also... should this become a configuration rather than a static setting? Maybe we address this another time.

Reply:
I'm going to revert this. I want to drop the 10min `ackTimeout` Orca sets on `RunTask` message and was thinking of keeping the length of time over which a message the system can't ack is retried unchanged, but I'm into making the configurable later and being more thoughtful with it.